<launch>
  <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" />

  <!-- Depth Cloud Only -->
  <!--
  <node pkg="nodelet" type="nodelet" name="nodelet1"
        args="load depth_image_proc/point_cloud_xyz nodelet_manager">
    <remap from="camera_info" to="/camera_throttled/depth/camera_info"/>
    <remap from="image_rect" to="/camera_throttled/depth/image_rect_raw"/>
    <remap from="points" to="/camera_throttled/depth/points"/>
  </node>
  -->

  <!-- RGB-D Cloud -->
  <node pkg="nodelet" type="nodelet" name="nodelet1"
        args="load depth_image_proc/point_cloud_xyzrgb nodelet_manager">
  <remap from="rgb/camera_info" to="/camera_throttled/color/camera_info"/>
  <remap from="depth_registered/image_rect" to="/camera_throttled/depth/image_rect_raw"/>
  <remap from="rgb/image_rect_color" to="/camera_throttled/color/image_raw"/>
  <remap from="depth_registered/points" to="/camera_throttled/depth/color/points"/>
  </node>
</launch>
